_topic_maps/_topic_map.yml

```diff
@@ -267,6 +267,12 @@ Topics:
     File: installing-bare-metal-network-customizations
   - Name: Installing a user-provisioned bare metal cluster on a restricted network
     File: installing-restricted-networks-bare-metal
+- Name: Installing on bare metal with Assisted Installer
```

Comment:
Do we purposefully want to limit the scope to bare metal?

Reply:
Currently isn't that the only available option for AI? We can't let users think they can use AI to install on any platform when it's only for bare metal. Down the line if that changes, then that's a discussion to have, but the scope IS limited at the moment...

Reply:
Can we restate it's on-prem vs bare metal?
because VMs on KVM / vsphere will work just fine and supported